Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -180,3 +188,41 @@ extension Navigator: ElectronicHorizonObserver {
         NotificationCenter.default.post(name: .electronicHorizonDidPassRoadObject, object: nil, userInfo: userInfo)
     }
 }
+
+extension Navigator: NavigatorObserver {
+    func onStatus(for origin: NavigationStatusOrigin, status: NavigationStatus) {
+        guard origin == .locationUpdate else { return }
```

Comment:
Other possible origins are `legChange`, `setRoute` and `unconditional`. 
I filter them here to save the same behavior as we had before.